.github/workflows/development.yml

```diff
@@ -23,7 +23,7 @@ jobs:
           node-version: 21.7.3
 
       - name: "Install dependencies"
-        run: npm ci
+        run: npm ci --include=optional
```

Comment:
Why is this needed?

Reply:
I think we need something like this in order to run the github actions now (to include the "nx-linux-x64-gnu" here and not locally) but I'm just going to double check it's necessary.

Reply:
Okay apparently I'm full of it it, might have been something else that needed this. Removed now!